### PR TITLE
updating blogs and presentation docs

### PR DESCRIPTION
**Pull request by leandroBorgesFerreira**
Adding one article

### CODE REVIEW

File: modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md [hunk 1]
```diff
@@ -158,8 +158,11 @@ April '18 - version 0.7
 
 ### Arrow 101 — Building an Android app using Functional Programming
 
-[Leandro Ferreira](https://twitter.com/mLeandroBF) presents how to build a simple app using Arrow, step by step. 
+[Leandro Ferreira](https://twitter.com/mLeandroBF) presents how to build a simple app using Arrow and how to implement
+solutions with Semigroups.
```
Comment (pakoito) [inline modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md]: There's an unnecessary line jump here. Ready to merge otherwise!

**Comment by raulraja** [comment]
what's up with codecov? The docs should have been ignored to calculate code coverage.

File: modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md (hunk 1 above)
Comment (leandroBorgesFerreira) [reply modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md]: Which line? 🤔

Comment (pakoito) [reply modules/docs/arrow-docs/docs/docs/quickstart/blogs/README.md]: Between implement and solutions. See the line indicator jumps from 161 to 162, even if the render looks okay.

**Comment by JorgeCastilloPrz** [comment]
There's something wrong with it, we've been having similar problems on [this one](https://github.com/arrow-kt/arrow/pull/998).

**Comment by pakoito** [comment]
AFAIK Codecov has been doing that since we re-enabled it. Even no-op PRs sometimes decrease coverage lol

**Pull request merged**